examples/src/charmory_examples/object_detection/xview/train_xview_model.py

```diff
@@ -0,0 +1,342 @@
+import albumentations as A
+
+import numpy as np
+import torch
+
+
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+#DEVICE = 'cpu'
+BATCH_SIZE = 8
+import torch
+
+torch.set_float32_matmul_precision("high")
+#import armory.data.datasets
+import datasets
+from datasets import load_from_disk
+from datasets.filesystems import S3FileSystem
+
+LABEL_MAP = {
```

Comment:
Can we add a comment documenting what we're mapping from and to? And why we have to remap the labels.

Reply:
I took this out, thanks. It was from when I was using the original xview dataset and there were 63 classes and the labels were not in order and skipped some numbers for some reason.